### ISSUE TITLE
Consolidate the instructions for local deployments with kind of minikube

### ISSUE THREAD

**Issue by AlanCoding** [ansible/tower-operator#1455]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx-operator/issues) for duplicates.
- [X] I understand that AWX Operator is open source software provided for free and that I might not receive a timely response.

### Feature Summary

The docs here mention minikube

https://github.com/ansible/awx-operator/blob/devel/README.md#creating-a-minikube-cluster-for-testing

However, @relrod has done a very good job over in AWX developing a more complete and set of up-to-date instructions.

https://github.com/ansible/awx/blob/devel/docs/development/kind.md

https://github.com/ansible/awx/blob/devel/docs/development/minikube.md

I have had the experience of beating my head against the table trying to follow the `README.md` here and struggling, in ignorance of the AWX docs.

My ask here is that the README here has a section that mainly defers to the AWX docs. If there is anything unique here that needs to be added, we should move that over.

Also, as a bonus @fosterseth has some content here:

https://gist.github.com/fosterseth/081c05248975048beb55858def010866

I would also like to merge that into the docs in AWX.

So to recap:
 - Merge `minikube` instructions in the awx-operator README into the AWX development docs
 - Delete the content in the awx-operator README, and replace with link to AWX docs
 - See if Seth's gist has any additional content to add to those AWX development docs

I believe that centralizing these instructions will help us moving forward, as I hear people still frequently have trouble with this.
